ozaria/site/components/interactive/PageInteractive/insertCode/index.vue

```diff
@@ -43,30 +39,33 @@
       const language = (this.codeLanguage || "").toLowerCase() === 'javascript' ? 'javascript' : 'python'
```

Comment:
This can be drive by a default prop value

Reply:
I think its there in ozariaUtils